stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/common/navigation/destinations/AttachmentDestination.kt

```diff
@@ -135,5 +167,10 @@ public open class AttachmentDestination(
 
     private companion object {
         private const val VIDEO_MIME_TYPE_PREFIX = "video"
+        private const val MP4_MIME_TYPE = "mp4"
```

Comment:
Are they going to fix that on their side? `mp4` is not a mime type , it is mime subtype.